src/main/java/duke/command/Duke.java

```diff
@@ -0,0 +1,247 @@
+                    Deadline deadline = new Deadline(text[2], text[3]);
+                    deadline.setDone(!text[1].equals(" "));
+                    tasks.add(deadline);
+                } else if (line.startsWith("E")) {
+                    Event event = new Event(text[2], text[3], text[4]);
+                    event.setDone(!text[1].equals(" "));
+                    tasks.add(event);
+                }
+            }
+        } catch (FileNotFoundException e) {
+            System.out.println("Something went wrong: " + e.getMessage());
+            throw new RuntimeException(e);
+        } catch (NoSuchElementException e) {
+            System.out.println("Sorry no line found. ");
+        }
+    }
+
+    public static void FileWriting(String filePath) {
+        //File f = new File(filePath);
+        try {
+            FileWriter fw1 = new FileWriter(filePath);
+            fw1.write("");
+            FileWriter fw2 = new FileWriter(filePath, true);
+            for (Task t: tasks) {
+                if (t instanceof duke.task.Todo) {
+                    Todo todo = (Todo) t;
+                    String text = "T|" + todo.getStatusIcon() + "|" + todo.getDescription() + "\n";
+                    fw2.write(text);
+                } else if (t instanceof duke.task.Deadline) {
+                    Deadline deadline = (Deadline) t;
+                    String text = "D|" + deadline.getStatusIcon() + "|" + deadline.getDescription() + "|" + deadline.getBy() + "\n";
+                    fw2.write(text);
+                } else if (t instanceof duke.task.Event) {
+                    Event event = (Event) t;
+                    String text = "E|" + event.getStatusIcon() + "|" + event.getDescription() + "|" + event.getFrom() + "|" + event.getTo() + "\n";
+                    fw2.write(text);
+                }
+            }
+            fw1.close();
+            fw2.close();
+            System.out.println("Your task list is saved!");
+        } catch (IOException e) {
+            System.out.println("Something went wrong: " + e.getMessage());
+            throw new RuntimeException(e);
+        }
+    }
+
+
+    public static void main(String[] args) throws IOException {
+
+        greeting();
+
+        File duke = new File("./duke.txt");
+        if (duke.exists()) {
+            System.out.println("Task list already exists.");
+            FileReading("./duke.txt");
+            list_tasks();
+        } else {
+            System.out.println("Your new task list is created!");
+        }
+        System.out.println("Please input your command: ");
+
+
+        while(true){
+            Scanner in = new Scanner(System.in);
+            String line = in.nextLine();
+            if(line.equals("bye")){
+                FileWriting("./duke.txt");
+                System.out.println("Bye. Hope to see you again soon!");
+                break;
+            }
+            else if(line.startsWith("mark")){
```

Comment:
Your way of handling commands is not ideal. Do try to think of better ways to parse/handle your commands. (Hint: Using `startsWith` is not really recommended)